### ISSUE TITLE
The latest version vscode1.93.0 cannot display preview

### ISSUE THREAD

**Issue by gnakilgnoh** [ZenUml/vscode-extension#5]
I think zenuml is very good, is it possible for the author to take the time to fix it?

![image](https://github.com/user-attachments/assets/9876d43e-28c6-49a1-b019-75490537bf7c)

**Comment by MrCoder** [comment]
Hi @gnakilgnoh, are you also using ZenUML on other platforms?

**Comment by gnakilgnoh** [comment]
> Hi @gnakilgnoh, are you also using ZenUML on other platforms?

Yes. I downloaded the windows version of the software so it can be used normally.
